databricks/koalas/indexes.py

```diff
@@ -422,6 +446,17 @@ def to_pandas(self) -> pd.MultiIndex:
 
     toPandas = to_pandas
 
+    def dropna(self):
+        """
+        Return MultiIndex without NA/NaN values
+        """
+        kdf = self._kdf
+        sdf = self._kdf._internal.sdf
+        for index_scol in self._kdf._internal.index_scols:
+            sdf = sdf.where(index_scol.isNotNull())
```

Comment:
I think you can make the condition with `and` and `reduce` as you did in other places.

Reply:
Thanks for reviewing! i just changed. i'll commit after resolve conflicts :)